hack/kpng-local-up.sh

```diff
@@ -39,7 +43,9 @@ function build {
 function install {
     # substitute it with your changes...
     echo "Applying template with KPNG_IMAGE=$IMAGE"
-    cat kpng-deployment-ds.yaml.tmpl | sed "s,KPNG_IMAGE,$IMAGE," > kpng-deployment-ds.yaml
+    cat kpng-deployment-ds.yaml.tmpl | sed "s,IMAGE,$IMAGE," > kpng-deployment-ds.yaml.1
```

Comment:
Just for simplicity, dont want to overabstract this yet.
someday we'll have ytt or kustomize or whatever here, or just have a go launcher